package.json

```diff
@@ -8,29 +8,39 @@
     "url": ""
   },
   "scripts": {
-    "start": "tsc && concurrently \"tsc -w\" \"lite-server\" "
+    "start": "tsc && concurrently \"tsc -w\" \"lite-server\" ",
+    "ngc": "ngc -p tsconfig-ngc.json",
+    "rollup": "rollup -c rollup-config.js",
+    "cleanup": "rm -rf _out dist"
   },
   "dependencies": {
-    "@angular/common": "2.0.0",
-    "@angular/compiler": "2.0.0",
-    "@angular/core": "2.0.0",
-    "@angular/forms": "2.0.0",
-    "@angular/http": "2.0.0",
-    "@angular/platform-browser": "2.0.0",
-    "@angular/platform-browser-dynamic": "2.0.0",
-    "@angular/router": "3.0.0",
-    "systemjs": "0.19.27",
+    "@angular/common": "^4.1.3",
```

Comment:
Angular2 and Angular4 are pretty similar I think - the big jump was from ng1 to ng2